flexmeasures/ui/templates/crud/account.html

```diff
@@ -1,183 +1,345 @@
+    <button
+      class="btn btn-sm btn-responsive btn-info m-4"
+      type="submit"
+      title="View history of user actions."
+    >
+      Audit log
+    </button>
+  </form>
 </div>
 <div class="container-fluid">
-    <div class="row">
-        <div class="col-md-2"></div>
-        <div class="col-md-8">
-            <div class="card">
-                <h3>Account</h3>
-                <small>Account: {{ account.name }}</small>
-                <div class="table-responsive">
-                    <table class="table table-striped table-responsive">
-                        <tbody>
-                            <tr>
-                                <td>
-                                    ID
-                                </td>
-                                <td>
-                                    {{ account.id }}
-                                </td>
-                            </tr>
-                            <tr>
-                                <td>
-                                    Roles
-                                </td>
-                                <td>
-                                    {{ account.account_roles | map(attribute='name') | join(", ") }}
-                                </td>
-                            </tr>
-                            {% if account.consultancy_account_name %}
-                            <tr>
-                                <td>
-                                    Consultancy
-                                </td>
-                                <td>
-                                    {{ account.consultancy_account_name }}
-                                </td>
-                            </tr>
-                            {% endif %}
-                        </tbody>
-                    </table>
+  <div class="row">
+    <div class="col-md-2">
+      <div class="sidepanel-container">
+        <div
+          class="left-sidepanel-label"
+          style="transform: translateX(-30%) !important"
+        >
+          Edit Account
+        </div>
+        <div class="sidepanel left-sidepanel">
+          <form class="form-horizontal" id="editaccount">
+            <fieldset>
+              <div class="asset-form">
+                <h3>Edit {{ account.name }}</h3>
+                <small
+                  >Owned by account: {{ account.name }} (ID: {{ account.id
+                  }})</small
+                >
+
+                <!-- AccountID field -->
+                <div class="form-group">
+                  <div class="col-md-3">
+                    <label for="consultant_account_id" class="control-label"
+                      >Consultant Account ID</label
+                    >
+                    <input
+                      type="text"
+                      class="form-control"
+                      id="consultant_account_id"
+                      name="account"
+                      value="{{ account.consultancy_account_id }}"
```

Comment:
If the value is `None`, we should not write `None`.

And this should show a dropdown of accounts I can choose from.

And having no consultancy account should be okay (then we should maybe not send this field at all, it remaining empty)

And only show this if I am an admin.